Python/pywarpx/picmi.py

```diff
@@ -1656,12 +1657,52 @@ def laser_antenna_initialize_inputs(self, laser):
 
 
 class LoadInitialField(picmistandard.PICMI_LoadGriddedField):
+    def init(self, kw):
+        self.do_divb_cleaning_external = kw.pop('warpx_do_divb_cleaning_external', True)
```

Comment:
Same comment as below.
```suggestion
        self.do_divb_cleaning_external = kw.pop('warpx_do_divb_cleaning_external', None)
```